cli/script/command-executor.ts

```diff
@@ -1175,7 +1175,9 @@ export var releaseCordova = (command: cli.IReleaseCordovaCommand): Promise<void>
                 throw new Error("Platform must be either \"ios\" or \"android\".");
             }
 
-            var cordovaCommand: string = command.build ? "build" : "prepare";
+            var cordovaCommand: string = command.build ? 
+                (command.isReleaseBuildType ? "build --release" : "build") :
```

Comment:
@silhouettes thank you for your feedback. I thought initially about this and I was a little bit confused about naming it `release `because it would be semantically closer to something related to our `code-push release` command which actually perform the release of our updates to code push server and has nothing in common with release build flag. So I think naming this flag as  `release` may lead to some confusion. If you don't mind I'd prefer to avoid 'release' name here in this particular case.